bootstrap.yml

```diff
@@ -14,7 +14,7 @@
       changed_when:
         - output.stdout != ""
         - output.stdout != "\r\n"
-      when: ansible_os_family == "Debian"
+      when: islandora_vagrant_distro == "ubuntu/xenial64"
 
```

Comment:
Then I think I would add a new task here to gather facts: 
```yaml
# Gathers facts, since it has to be disabled for ubuntu above
- name: Gather Facts
  setup:
```